ci/terraform/oidc/reset-password-request.tf

```diff
@@ -0,0 +1,51 @@
+module "reset-password-request" {
```

Comment:
You'll need to add the triggers and the depends_on to the oidc/api-gateway.tf

Reply:
added